File: repos_estudiantes_TARQUI_tercera_entrega/ILLANES_ESCOBAR_FERNANDO_JOHSUA/PERSISTENCIA/Ejercicio3/Java/Main.java
package PERSISTENCIA.Ejercicio3.Java;

public class Main {
    public static void main(String[] args) {
        GestorCliente gestor = new GestorCliente();

        Cliente c1 = new Cliente(1, "Bruno", "71511111");
        Cliente c2 = new Cliente(2, "Paola", "76543210");

        gestor.guardarCliente(c1);
        gestor.guardarCliente(c2);

        System.out.println("Buscar cliente con ID 2:");
        System.out.println(gestor.buscarCliente(2));

        System.out.println("Buscar celular del cliente con ID 1:");
        System.out.println(gestor.buscarCelularCliente(1));
    }
}

File: repos_estudiantes_TARQUI_tercera_entrega/ILLANES_ESCOBAR_FERNANDO_JOHSUA/PERSISTENCIA/Ejercicio3/Python/main.py
from cliente import Cliente
from gestor_cliente import GestorCliente

gestor = GestorCliente()

c1 = Cliente(1, "Bruno", "71511111")
c2 = Cliente(2, "Paola", "76543210")

gestor.guardar_cliente(c1)
gestor.guardar_cliente(c2)

print("Buscar cliente con ID 2:")
print(gestor.buscar_cliente(2))

print("Buscar celular del cliente con ID 1:")
print(gestor.buscar_celular_cliente(1))
